templates/base/storybook/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "storybook",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook -c .storybook -o ./out"
+  },
+  "dependencies": {
+    "@babel/core": "^7.9.6",
+    "@babel/preset-react": "^7.9.4",
+    "@storybook/addon-a11y": "^5.3.18",
+    "@storybook/addon-actions": "^5.3.18",
+    "@storybook/addon-docs": "^5.3.18",
+    "@storybook/addon-jest": "^5.3.18",
+    "@storybook/addon-knobs": "^5.3.18",
+    "@storybook/addon-links": "^5.3.18",
+    "@storybook/addon-storysource": "^5.3.18",
+    "@storybook/addon-viewport": "^5.3.18",
+    "@storybook/react": "^5.3.18",
+    "babel-loader": "^8.1.0",
+    "css-loader": "^3.5.3",
+    "react": "^16.13.1",
```

Comment:
Do we need following? This will be imported in components 

"react": "^16.13.1",
    "react-dom": "^16.13.1",

Reply:
@vinodloha - Storybook expects them as peerDependencies but removing them working in my case.